lib/glific/flows/flow.ex

```diff
@@ -375,6 +375,7 @@ defmodule Glific.Flows.Flow do
         &Node.validate(&1, &2, flow)
       )
       |> dangling_nodes(flow)
+      |> check_flow_keywords(flow)
```

Comment:
@AkhileshNegi please take care of the consistency. We already have a validation structure in place. 
There should be a function called Wait.validate just like Node. validate and if it's not there please create one.

Reply:
@pankaj-ag Thanks for pointing out. I have restructured the code now